src/algebra/group/hom.lean

```diff
@@ -99,18 +143,22 @@ end is_monoid_hom
 
 namespace is_add_monoid_hom
 
-instance is_add_monoid_hom_mul_left {γ : Type*} [semiring γ] (x : γ) : is_add_monoid_hom (λ y : γ, x * y) :=
+/-- left multiplication in a ring is an additive monoid morphism -/
+instance is_add_monoid_hom_mul_left {γ : Type*} [semiring γ] (x : γ) :
+  is_add_monoid_hom (λ y : γ, x * y) :=
 { map_zero := mul_zero x, map_add := λ y z, mul_add x y z }
 
-instance is_add_monoid_hom_mul_right {γ : Type*} [semiring γ] (x : γ) : is_add_monoid_hom (λ y : γ, y * x) :=
+/-- right multiplication in a ring is an additive monoid morphism -/
+instance is_add_monoid_hom_mul_right {γ : Type*} [semiring γ] (x : γ) :
+  is_add_monoid_hom (λ y : γ, y * x) :=
 { map_zero := zero_mul x, map_add := λ y z, add_mul y z x }
 
-
 end is_add_monoid_hom
 
-/-- Predicate for group homomorphism. -/
+/-- Predicate for group homomorphisms. -/
```

Comment:
Shouldn't this also get a deprecation warning?

Reply:
OK!